### PR TITLE
2696 cases in redis cache

### PR DESCRIPTION
**Pull request by trystant**
In your PR did you:

  - [X] Include a description of the changes? Caches the maps query so that there is only one hit per redis server restart rather than a database request every time the page is loaded. 
  - [X] Mention the issue the PR addresses?  This will significantly reduce the page load times after the initial page load
  - [ ] Include screenshots of any changes to the UI? N/A
  - [X] Add and/or update specs for your code?

### CODE REVIEW

File: app/controllers/cases_controller.rb [hunk 1]
```diff
@@ -114,7 +115,7 @@ def undo
       end
       flash[:success] = 'Undid that!'
       flash[:undo] = @this_case.versions
-    rescue
+    rescue StandardError
```
Comment (rlgreen91) [inline app/controllers/cases_controller.rb]: Let's not rescue all of `StandardError`

Comment (trystant) [reply app/controllers/cases_controller.rb]: OK. This can be modified to handle the specific sets of errors expected. Does that work better @rlgreen91 ?

Comment (rlgreen91) [reply app/controllers/cases_controller.rb]: Yes, it does.

**Comment by rlgreen91** [comment]
Is this good for me to merge @trystant ?

**Pull request merged**